### DISCUSSION TITLE
Is there a way to preselect a filter?

### DISCUSSION THREAD

**Discussion by eswachsman** [rappasoft/laravel-livewire-tables#280] [Q&A] [answered]
Is there a way to preselect a filter when the page loads? I have a user filter and I want it by default to be filtered to the current user. Is there a way to do that?

**Comment by rappasoft** [comment, marked as answer]
https://github.com/rappasoft/laravel-livewire-tables/wiki/Setting-a-default

Just wrote this so don't think you missed it.

**Reply by eswachsman** [reply]
You are so awesome! Thanks!
